client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,139 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { SitesTable } from './sites-table';
+
+const EmptyContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	height: 600px;
+
+	.empty-content__line {
```

Comment:
Using nested selectors breaks modularity. Imagine you'd want to use this component but don't apply this line styling to a single instance of it. How would you do it? 🙂

So, my suggestion is to create a separate component (`styled.p`), add the `max-width` property there and use this component inside the `line` prop in `<EmptyContent />` instead.

Reply:
Thanks for insight again, makes a lot of sense, will update accordingly